### ISSUE TITLE
JoinTable decorator nullable

### ISSUE THREAD

**Issue by harishajdarevic** [typeorm/typeorm#1094]
We have one table which connect two other tables with two @ManyToOne decorators.

One of decorator looks like this:
```
  @ManyToOne(type => Files, file => file.fileTag)
  @JoinColumn({nullable: false})
  file: Files;

  @ManyToOne(type => Tags, tag => tag.fileTag)
  @JoinColumn({nullable: true})
  tag: Tags;
```

Nullable option seems that doesn't work. Are we doing something wrong or ?

We are using version 0.1.1

**Comment by pleerock** [comment]
`@JoinColumn` does not have `nullable` option. You can define nullable option this way:

```ts
  @ManyToOne(type => Tags, tag => tag.fileTag, { nullable: true })
  @JoinColumn()
  tag: Tags;
```

however, there are some problems with nullable (relations only) that are planned to be fixed in `0.2.0`

**Comment by pleerock** [comment]
I can confirm all previously issues with nullable relations I knew about are fixed in recent 0.2.0-alpha versions. Did you found a solution working for you? Let me know if you did not.
